UI/CombatReport/CombatLogWnd.cpp

```diff
@@ -309,7 +304,7 @@ namespace {
 
 }
 
-LinkText * CombatLogWnd::Impl::DecorateLinkText(std::string const & text) {
+std::shared_ptr<LinkText> CombatLogWnd::Impl::DecorateLinkText(std::string const & text) {
```

Comment:
prefer no space between `const` and `&` (not originating in this change, I realize)